databricks/koalas/frame.py

```diff
@@ -2598,6 +2607,25 @@ def describe(self) -> 'DataFrame':
         mean         2.0       5.0
         stddev       1.0       1.0
         min          1.0       4.0
+        25%          1.0       4.0
+        50%          2.0       5.0
+        75%          3.0       6.0
+        max          3.0       6.0
+
+        Describing a ``DataFrame`` and selecting custom percentiles.
+
+        >>> df = ks.DataFrame({'numeric1': [1, 2, 3],
+        ...                    'numeric2': [4.0, 5.0, 6.0]
+        ...                   },
+        ...                   columns=['numeric1', 'numeric2', 'object'])
```

Comment:
It sould be  `columns=['numeric1', 'numeric2']`

Reply:
done